### PR TITLE
Return empty table for issue #1836

### PR DESCRIPTION
**Pull request by HarrietAkot**
Hello @bsipocz . I am an outreachy applicant. This is my first contribution to open source. Following the issue #1836 . This is my proposed solution. It should be able to return an empty table.

### CODE REVIEW

File: astroquery/sha/core.py [hunk 1]
```diff
@@ -145,6 +145,9 @@ def parse_line(line, cs=cs):
     # To table
     # transpose data for appropriate table instance handling
     t = Table(list(zip(*data)), names=col_names, dtype=dtypes)
+    if len(data) < 0:
+        warnings.warn(NoResultsWarning("No matching rows were found"))
```
Comment (daisycodes) [inline astroquery/sha/core.py:149]: Hi @HarrietAkot, nice job on your first PR. Can you clarify this to me? I see your using `warnings` and `NoResultsWarnings` but I don't see where they're defined or imported. See this sample file https://github.com/astropy/astroquery/blob/e49a63b9947e448ae71ee08c5261adca283d0ae7/astroquery/mast/cutouts.py. You can search for `warnings` and `NoResultsWarning` and you'll see how they're imported.

**Review by daisycodes — COMMENTED** [review]
Nice work @HarrietAkot. Left a comment. Maybe a test for this? I mean since this is changing the logic?

**Comment by HarrietAkot** [comment]
Thank you for the feedback on my PR. I will implement them as soon as
possible.

**Comment by daisycodes** [comment]
@HarrietAkot on a second thought, it would seem someone else is working on that ticket. See this https://github.com/astropy/astroquery/pull/1837. Did you check to make sure they've abandoned it?

**Comment by HarrietAkot** [comment]
Hello. Unfortunately this was the only package novice issue I was able to
find and take on. The other package novice issue was from 2018 and from
what I read on the thread it was labeled package novice but is a bit
advanced for beginners.

**Comment by bsipocz** [comment]
I would suggest to close this PR. There are a few issues with the suggested change, and the other one @daisycodes mentions predates this a tiny bit and is closer to get merged.

Nevertheless, thank you @HarrietAkot for working on this! Your other PR I'm sure will get merged soon.

File: astroquery/sha/core.py (hunk 1 above)
Comment (bsipocz) [reply astroquery/sha/core.py:149]: Thank you @daisycodes for making code review!

**Pull request closed without merging**